crates/control/src/behavior/stand.rs

```diff
@@ -50,13 +51,32 @@ pub fn execute(
                 .ball
                 .map(|state| state.ball_in_ground)
                 .unwrap_or(fallback_target);
-            Some(MotionCommand::Stand {
-                head: HeadMotion::LookAt {
-                    target,
-                    image_region_target: Default::default(),
-                    camera: None,
-                },
-            })
+            match (role, world_state.filtered_game_controller_state) {
+                (
+                    Role::Keeper,
+                    Some(FilteredGameControllerState {
+                        sub_state: Some(SubState::PenaltyKick),
+                        ..
+                    })
+                    | Some(FilteredGameControllerState {
+                        game_phase: GamePhase::PenaltyShootout { .. },
+                        ..
+                    }),
+                ) => Some(MotionCommand::ArmsUpStand {
+                    head: HeadMotion::LookAt {
+                        target,
+                        image_region_target: Default::default(),
+                        camera: None,
+                    },
+                }),
+                _ => Some(MotionCommand::Stand {
+                    head: HeadMotion::LookAt {
+                        target,
+                        image_region_target: Default::default(),
+                        camera: None,
+                    },
+                }),
+            }
```

Comment:
so doch net... stick to the variables you had, but don't make them mutable and assign them using a match statement at creation

Reply:
Doughne